index.bs

```diff
@@ -232,6 +232,24 @@ request could be rejected or even two Picture-in-Picture windows could be
 created. Regardless, the User Agent will have to fire the appropriate events
 in order to notify the website of the Picture-in-Picture status changes.
 
+# Arbitrary Content Specific Concepts # {#arbitrary-concepts}
+
+The Picture-in-Picture window MAY support <dfn>arbitrary content</dfn> which will
+display a {{Document}} in the Picture-in-Picture window. The <a>arbitrary content</a>
+MAY be <dfn>interactive arbitrary content</dfn> which will allow <dfn>user
+interaction</dfn>. This includes touch, gesture, mouse and keyboard user input
+events. <!-- TODO(beccahughes): Expand this as part of security considerations -->
+
+The <a>arbitrary content</a> MAY also be <dfn>non-interactive arbitrary content</dfn>
+which will NOT allow <a>user interaction</a>. However, the user agent MAY show
+Picture-in-Picture window controls to support some interaction.
```

Comment:
Shall we explicitly say that those controls can be a close button. back to tab, etc?
And maybe link to the Media Session API section.

Reply:
Done